lib/preboot/logger.js

```diff
@@ -9,6 +9,11 @@ var debug = require('diagnostics')('slay:preboot:logger');
  * a winston Logger for this app instance.
  */
 module.exports = function (app, options, callback) {
+  if (app.log) {
```

Comment:
This capability checking is not quite enough as there are a few calls to `app.log.info` in the other preboots.